domain/goal/src/test/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/domain/goal/GoalTest.kt

```diff
@@ -22,7 +22,8 @@ class GoalTest {
     val goal = aValidGoal(steps = mutableListOf(step2, step3, step1)) // Steps passed into the goal out of sequence
 
     // Then
-    assertThat(goal.steps.map { it.title }).containsExactly( // steps are returned in correct sequence based on sequenceNumber
+    assertThat(goal.steps.map { it.title }).containsExactly(
```

Comment:
This is the biggest (and most annoying IMHO) change in this PR.
Comments in a value list must now be on their own line, so everywhere we originally had a comment after the statement (if it's in a list of things), it now needs to be on it's own line! 😢 

For reference, this is the linting error message:
```
A comment in a 'value_argument_list' is only allowed when placed on a separate line (cannot be auto-corrected)
```
Theres some chat about it here: https://github.com/pinterest/ktlint/issues/2445